### PR TITLE
Fixes #25447: Admin user should have notification that their session is kept until timeout

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/25447

### CODE REVIEW

File: user-management/src/main/elm/sources/UserManagement.elm [hunk 1]
```diff
@@ -322,6 +319,24 @@ update msg model =
             in
                 ({model | ui = newUI, userForm = newUserForm}, Cmd.none)
 
+-- When a with user with admin rights  is updated, changes will not have effect now: session is kept until it expires
```
Comment (fanf) [inline user-management/src/main/elm/sources/UserManagement.elm:322]: `when a user`|

**Comment by clarktsiory** [comment]
PR updated with a new commit

File: user-management/src/main/elm/sources/UserManagement.elm [hunk 2]
```diff
@@ -322,6 +319,24 @@ update msg model =
...
+    let
+        hasAdminRights =
+            model.users
+                |> Dict.get username
+                |> Maybe.map (.authz)
+                |> Maybe.map (List.any (\auth -> "any_rights" == auth || String.startsWith "administration_" auth))
+                |> Maybe.withDefault False
+        message =
+            if hasAdminRights then
+                username ++ " has been modified, change will be taken into account on next startup"
```
[the remark this answers is not in the document]
Comment (clarktsiory) [reply user-management/src/main/elm/sources/UserManagement.elm]: yes ! next user login !

**Review by fanf — APPROVED** [review]

**Comment by clarktsiory** [comment]
OK, squash merging this PR

**Comment by clarktsiory** [comment]
OK, merging this PR

**Pull request merged**